js/market.js

```diff
@@ -1,59 +1,71 @@
 
 $.get('http://localhost:5000/plans')
     .done((result) => {
-        // let product=JSON.stringify(result);
-        let f = document.getElementById('form-box');
+        console.log(result);

```

Comment:
remember- the client sees everything.
do not log app info into the client console.